packages/components/src/internal/components/domainproperties/list/ListPropertiesAdvancedSettings.test.tsx

```diff
@@ -20,6 +20,14 @@ const emptyNewModel = ListModel.create(null, DEFAULT_LIST_SETTINGS);
 const populatedExistingModel = ListModel.create(getDomainDetailsJSON);
 
 describe('AdvancedSettings', () => {
+    beforeEach(() => {
+        LABKEY.moduleContext = {
+            core: {
+                deprecatedObjectLevelDiscussions: false,
```

Comment:
could add a quick test for the `deprecatedObjectLevelDiscussions: true` case as well...but given that it is a feature going away soon it isn't high priority.